tcms/testruns/templates/testruns/get.html

```diff
@@ -119,9 +119,12 @@ <h2 class="card-pf-title" style="text-align: left">
 
 <script src="{% static 'typeahead.js/dist/typeahead.jquery.min.js' %}"></script>
 <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
+<script src="{% static 'moment/moment.js' %}"></script>
+<script src="{% static 'moment-timezone/moment-timezone.js' %}"></script>
 
 <script src="{% static 'js/utils.js' %}"></script>
 <script src="{% static 'js/jsonrpc.js' %}"></script>
 <script src="{% static 'js/tags.js' %}"></script>
+<script src="{% static 'moment-timezone/builds/moment-timezone-with-data.min.js' %}"></script>
```

Comment:
is there a reason this needs to be here instead of groupped with the rest of the moment.js static files ?

Reply:
not anymore.